pkg/scheduler/controller/shoot/scheduler_control.go

```diff
@@ -129,7 +128,7 @@ func (c *defaultControl) ScheduleShoot(ctx context.Context, obj *gardencorev1bet
 	// If no Seed is referenced, we try to determine an adequate one.
 	seed, err := determineSeed(shoot, c.seedLister, c.shootLister, c.cloudProfileLister, c.config.Schedulers.Shoot.Strategy)
 	if err != nil {
-		c.reportFailedScheduling(shoot, err)
+		schedulerLogger.Errorf("Scheduling Failed: Failed to determine Seed for Shoot %+v: %v", shoot, err)
```

Comment:
I think, `SchedulingFailed` should definitely be reported, as the event is the only interface through which the user can know, why his Shoot is not being created. (kube-scheduler also reports a similar event when it fails to schedule Pods)
Logging it makes sense, but let's keep the event.